### DISCUSSION TITLE
Struggling to configure RabbitMQ for TLS-PSK to enable connection from an IoT device using MQTT TLS-PSK

### DISCUSSION THREAD

**Discussion by sreenicse2009** [rabbitmq/rabbitmq-server#11505] [Questions] [answered]
Unable to find SSL configuration variables in RabbitMQ configuration for setting up 'TLS-PSK'

**Comment by lukebakken** [comment, marked as answer]
Hello,

You have not provided nearly enough information for anyone to assist you. In addition, only commercial users of RabbitMQ will receive support for TLS-related questions ([support policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md#exceptions-question-that-will-be-ignored)).

Since RabbitMQ uses the Erlang VM, this example project may demonstrate the settings you're looking for - https://github.com/zmstone/erlang-psk-example
